appveyor.yml

```diff
@@ -0,0 +1,40 @@
+skip_branch_with_pr: true
+
+image:
+  - Visual Studio 2017
+
+cache:
+  - C:\msys64 -> scripts/msys2-install-dependencies.sh
+
+init:
+  - echo "init"
+#  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```

Comment:
Perhaps add a comment that this is for Remote Desktop debugging?

Reply:
added comment